### ISSUE TITLE
Gitcoin Grants Feedback

### ISSUE THREAD

**Issue by vs77bb** [gitcoinco/web#6864]
- [ ] Missing Grants In Tech Category, infinite scrolling ends. i know bc i dont see Prysmatic when i get thru the list...but they are listed as [Tech in Admin](https://gitcoin.co/_administrationgrants/grant/24/change/)

![image](https://user-images.githubusercontent.com/23297747/84696312-4d5a5e80-af1a-11ea-86bf-12e4a1f83138.png)

- [x] Hide all Health Grants, even from ['All Categories'](https://gitcoin.co/grants/) 

![image](https://user-images.githubusercontent.com/23297747/84671610-96e48280-aef5-11ea-92b1-5cc4022ac7d8.png)

- [x] says rd 7 coming soon — should just show rd6 donors
![image](https://user-images.githubusercontent.com/23297747/84671657-a368db00-aef5-11ea-853a-671b953f46ce.png)

- [x] can't change graant categories in admin
![image](https://user-images.githubusercontent.com/23297747/84671810-cd220200-aef5-11ea-9051-9c0cd1bb2a5d.png)

- [x] phone verification currently busted
![image](https://user-images.githubusercontent.com/23297747/84671943-f478cf00-aef5-11ea-9e26-caf7329c9e1b.png)

- [ ] add to cart UX idea from scoott: we may want to at some fixed number shorten this to you funded 65 grants "x, y, z, a, b and 60 others"
![image](https://user-images.githubusercontent.com/23297747/84672087-1e31f600-aef6-11ea-9c23-6e4a0c324137.png)

- [ ] need a URL directly for the #CryptoForBlackLives category (maybe https://gitcoin.co/grants/?type=change&category=cryptoforblacklives) -- currently no hyperlink showing.. 

![image](https://user-images.githubusercontent.com/23297747/84681064-994cd980-af01-11ea-8f5c-43cb32b61fb9.png)

**Comment by willsputra** [comment]
- [ ] the "Verify my number" button should be changed to "Number verified ✅" after I've verified my number. at first glance I was slightly confused and wasn't sure whether my number was successfully verified or not.

![Screenshot 2020-06-16 16 37 26](https://user-images.githubusercontent.com/19514207/84752093-e6a77480-afef-11ea-9969-778b9afc3291.png)

**Comment by vs77bb** [comment]
- [ ] No sortby round 6 
- [ ] some copy typos to fix in there  

![image](https://user-images.githubusercontent.com/23297747/84773647-b4bfef00-afaa-11ea-976a-dbdd34877c8c.png)

**Comment by vs77bb** [comment]
- [ ] Did we remove public flags / negative voting?
